actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -737,6 +745,19 @@ public static void upsertUserOrgData(Map<String, Object> userMap, RequestContext
     return userDetails;
   }
 
+  private static void updateAllTncAcceptance(Map<String, Object> allTncAccepted) {
```

Comment:
update what to what?
I think jsonNode or objectNode could make the code much readable for such places.

Reply:
Update code `convertTncJsonStringToMapObject(Map<String, Object> allTncAccepted)`